src/systemverilog/systemverilog.ts

```diff
@@ -16,7 +16,29 @@ export const conf: IRichLanguageConfiguration = {
 	brackets: [
```

Comment:
i see a second brackets section on line 69. how do those play together?

Reply:
I think line 69 are for delimiters only and we don't need those begin* end* here.
also, it can create false match when someone has a signal name like:
assign **bus_begin_index** = blah << foo;

Reply:
gotcha makes sense